### PR TITLE
GTK 3.20 :: Fixed #430

### LINKED ISSUE #430: GTK 3.20 :: Icon selected background color backdrop mode
Not background color:
anim

Application: Nemo file selected, gtk3-widgets-factory

### PR DESCRIPTION
**Pull request by megax**
Global fixed.

### CODE REVIEW

**Comment by megax** [comment]
@lots0logs @khurshid-alam Fixed backdrop bug. Finally... :)

**Comment by megax** [comment]
It's pretty annoying bug, if this could be you should merge. (i massive tested)

File: gtk-3.20/scss/widgets/_misc.scss [hunk 1]
```diff
@@ -301,6 +301,7 @@
             &:disabled { color: mix($selected_fg_color, $selected_bg_color, .5); }
 
             &:backdrop {
+                background-color: $selected_bg_color; // Fixed Issue #430
```
Comment (satya164) [inline gtk-3.20/scss/widgets/_misc.scss]: We should add a `$backdrop_selected_bg_color` variable, which can be a faded out version.

Comment (megax) [reply gtk-3.20/scss/widgets/_misc.scss]: What colors can write to be prepared?

Comment (satya164) [reply gtk-3.20/scss/widgets/_misc.scss]: @megax I think we can keep it same as `$selected_bg_color` for this PR, and later change it.

Comment (megax) [reply gtk-3.20/scss/widgets/_misc.scss]: Okay.

**Comment by megax** [comment]
@satya164 Done.

**Comment by satya164** [comment]
Let's wait for @lots0logs

**Comment by khurshid-alam** [comment]
@megax Looks good. Please rebase your changes.

**Comment by megax** [comment]
@khurshid-alam Rebased.

**Pull request merged**